irohad/ametsuchi/mutable_storage.hpp

```diff
@@ -19,8 +20,11 @@ namespace shared_model {
 }  // namespace shared_model
 
 namespace iroha {
+  struct LedgerState;
+
   namespace ametsuchi {
 
+    class BlockStorage;
```

Comment:
Not including `BlockStorage` leads to compilation failures when `Result` is instantiated.